tests/test_registry.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
It would be good to to have a test to demonstrate the expected behavior when calling `Register` twice.

Reply:
Added one. It tests the opposite of what https://github.com/jadahl/xdg-desktop-portal/commits/wip/redundant-register expects.